### ISSUE TITLE
Android - Migrate Post list view setting

### ISSUE THREAD

**Issue by antonis** [wordpress-mobile/WordPress-Android#17827]
### Expected behavior
The view setting (list view or card view) in the posts list is migrated in the Jetpack app.

### Actual behavior
The view setting (list view or card view) in the posts list is not migrated in the Jetpack app.

### Steps to reproduce the behavior
1. Uninstall all WordPress and Jetpack apps on your device
2. Install the WordPress app
3. Login into the WordPress app
4. Go to your posts list and change the view setting from the button at the top-left
5. Go through the migration flow
6. **Notice** that the view setting (list view or card view) in the posts list is not migrated

**Comment by mkevins** [comment]
I was not able to reproduce this. 🤔 I assume that for this step:

> 4. Go to your posts list and change the view setting from the button at the top-left

that "top-left" was meant to be "top-right" (or was this performed in an RTL locale?).

I began looking into the underlying data, and it seems that the relevant flag is [already being migrated](https://github.com/wordpress-mobile/WordPress-Android/blob/b58b3e4fb3a5091d8c553d6d8e8b1c0d932c8e43/WordPress/src/main/java/org/wordpress/android/localcontentmigration/UserFlagsProviderHelper.kt#L62). I was curious why it might not be working as expected, but when I tested this flow, it _did_ work as expected:

https://user-images.githubusercontent.com/8507675/215394521-0af72627-3ee7-4747-9ecb-d7a0accf4390.mp4

I wonder what could be different about our steps that might result in different behavior 🤔 .
